packages/react-vapor/src/components/tooltip/tests/DateTooltip.spec.tsx

```diff
@@ -0,0 +1,42 @@
+import {ShallowWrapper} from 'enzyme';
+import {shallowWithState} from 'enzyme-redux';
+import * as moment from 'moment';
+import * as React from 'react';
+import * as _ from 'underscore';
+import {DateTooltip, DateTooltipsProps} from '../DateTooltip';
+import {Tooltip} from '../Tooltip';
+
+describe('Tooltip', () => {
+    let tooltipWrapper: ShallowWrapper<DateTooltipsProps>;
+    const TOOLTIP_PROPS: DateTooltipsProps = {
+        date: moment(),
+    };
+    const isoDate = moment().format();
+    const stringFormat = 'LLL';
+
+    describe('<DateTooltip />', () => {
+        beforeEach(() => {
+            tooltipWrapper = shallowWithState(<DateTooltip {...TOOLTIP_PROPS} />, {});
```

Comment:
The DateTooltip is not connected to redux, so no need to shallow with a state.
```suggestion
            tooltipWrapper = shallow(<DateTooltip {...TOOLTIP_PROPS} />);
```